src/framework/musesampler/internal/musesamplerwrapper.cpp

```
@@ -286,16 +284,29 @@ ms_Track MuseSamplerWrapper::addTrack()
-msecs_t MuseSamplerWrapper::playbackPosition() const
+muse::audio::TimePosition MuseSamplerWrapper::playbackPosition() const
 {
-    return samplesToMsecs(m_currentPosition, m_outputSpec.sampleRate);
+    return m_currentPosition;
 }
 
-void MuseSamplerWrapper::setPlaybackPosition(const msecs_t newPosition)
+void MuseSamplerWrapper::setPlaybackPosition(const muse::audio::TimePosition& position)
 {
-    m_sequencer.setPlaybackPosition(newPosition);
+    m_sequencer.setPlaybackPosition(position.time() * 1000.0); // secs to msecs
```

Cppcheck (2.20.0)

[style] The function 'version' is never used.

(unusedFunction)